cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -0,0 +1,38 @@
+
+{# ==========================================================================
+
+   table.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a gray well organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Well
+
+   table.row
+
+
+   ========================================================================== #}
+
+{% macro render(thead, rows) %}
+<table class="table__stack-on-small table__entry-header-on-small">
```

Comment:
As mentioned on the `list` topic, it's not actually needed. It's a "ghost block" of sorts, that only serves to be the basis for the necessary modifiers.

@KimberlyMunoz I agree, it'd be good to mention this scenario [where we talk about our BEM style](https://github.com/cfpb/front-end/blob/master/css.md#naming-conventions)